examples/identicon/bundle.js

```diff
@@ -1,16 +1,35 @@
-() => (function(){function r(e,n,t){function o(i,f){if(!n[i]){if(!e[i]){var c="function"==typeof require&&require;if(!f&&c)return c(i,!0);if(u)return u(i,!0);var a=new Error("Cannot find module '"+i+"'");throw a.code="MODULE_NOT_FOUND",a}var p=n[i]={exports:{}};e[i][0].call(p.exports,function(r){var n=e[i][1][r];return o(n||r)},p,p.exports,r,e,n,t)}return n[i].exports}for(var u="function"==typeof require&&require,i=0;i<t.length;i++)o(t[i]);return o}return r})()({1:[function(require,module,exports){
-
+() => (function () {
+  function r(e, n, t) {
+    function o(i, f) {
+      if (!n[i]) {
+        if (!e[i]) {
+          const c = typeof require === 'function' && require; if (!f && c) {
+            return c(i, !0);
+          } if (u) {
+            return u(i, !0);
+          } const a = new Error(`Cannot find module '${i}'`); throw a.code = 'MODULE_NOT_FOUND', a;
+        } const p = n[i] = { exports: {} }; e[i][0].call(p.exports, function (r) {
+          const n = e[i][1][r]; return o(n || r);
+        }, p, p.exports, r, e, n, t);
+      } return n[i].exports;
+    } for (var u = typeof require === 'function' && require, i = 0; i < t.length; i++) {
+      o(t[i]);
+    } return o;
+  } return r;
+})()({
+  1: [function (require, module, exports) {
 
     wallet.registerRpcMessageHandler(async (originString, requestObject) => {
-      console.log('hello', originString)
-      console.log(requestObject)
+      console.log('hello', originString);
+      console.log(requestObject);
       switch (requestObject.method) {
         case 'setUseBlockie':
-          return wallet.setUseBlockie(requestObject.params[0])
-        break
+          return wallet.setUseBlockie(requestObject.params[0]);
+          break;
         default:
-          throw new Error('Method not found. From index.js')
+          throw new Error('Method not found. From index.js');
       }
-    })
-    
-    },{}]},{},[1])
\ No newline at end of file
+    });
+
+  }, {}],
+}, {}, [1]);
```

Comment:
This and all other `examples/*/bundle.js` files should be deleted. Use `git rm --cached PATH_TO_FILE` and then commit the results. Once removed, they should already be ignored due to the `.gitignore` file. I must've missed this one (and any others).